composer.json

```diff
@@ -1,18 +1,20 @@
 {
     "name": "keboola/billing-api-php-client",
+    "description": "PHP Client for Keboola Connection Billing API",
     "license": "MIT",
     "require": {
         "php": ">=7.4",
         "ext-json": "*",
         "keboola/storage-api-client": "^13.0|^14.0",
-        "php-parallel-lint/php-parallel-lint": "^1.3",
-        "symfony/validator": "^2.3|^3.4|^4.2|^5.0"
+        "symfony/validator": "^5.0|^6.0"
     },
     "require-dev": {
-        "keboola/coding-standard": "^13.0",
-        "phpstan/phpstan": "^0.12.98",
-        "phpunit/phpunit": "^5.7",
-        "squizlabs/php_codesniffer": "^3.6"
+        "keboola/coding-standard": ">=14.0",
+        "phpstan/phpstan": "^1.8",
+        "phpstan/phpstan-phpunit": "^1.1",
+        "phpstan/phpstan-symfony": "^1.2",
```

Comment:
:thinking: tohle neni Symfony projekt

Reply:
no já už to dávám mechanicky všude, protože to fixuje i symfony/config, ale je fakt, že zrovna tady ho nemáme, tak to vyhodím